test/test_torch.py

```diff
@@ -6548,9 +6548,11 @@ def inner(self, device, dtype):
     ('true_divide', 'tensor_with_inplace', _small_3d,
         lambda t, d: [_small_3d(t, d, has_zeros=False)], 1e-1,
         1e-1, 1e-5, torch.testing.get_all_fp_dtypes()),
-    ('floor_divide', '', _small_3d, lambda t, d: [_number(3.14, 3, t)], 1, 1e-5, 1e-5, _types),
+    ('floor_divide', '', _small_3d, lambda t, d: [_number(3.14, 3, t)], 1, 1e-5, 1e-5, _types, (), True,
```

Comment:
Let's implement a floor_divide OpInfo and remove these.

Reply:
Done. Had to implement a new `supports_autograd` attribute to `OpInfo` since `floor_divide` isn't currently differentiable.

Reply:
That sounds like good metadata.